src/js/va-letters/utils/helpers.js

```diff
@@ -1,4 +1,40 @@
+import merge from 'lodash/fp/merge';
+
+import environment from '../../common/helpers/environment';
+
 export const chapterNames = {
   confirmAddress: 'Confirm Address',
   downloadLetters: 'Download Letters'
 };
+
+function isJson(response) {
```

Comment:
Yes, it's copied in a [bunch of places](https://github.com/department-of-veterans-affairs/vets-website/search?p=2&q=apiRequest&type=&utf8=%E2%9C%93): health-records, messaging, rx, and I agree it might make sense to share instead of duplicating the code (though the function signature does vary slightly across those apps).  Not sure this PR is the place to do that though...we can ask around on #vg-frontend or at the next FE architecture syn.